packages/config/src/projects/intmax/intmax.ts

```diff
@@ -21,14 +36,160 @@ export const intmax: ScalingProject = underReviewL2({
       ],
       repositories: ['https://github.com/InternetMaximalism'],
       socialMedia: ['https://twitter.com/intmaxIO'],
+      bridges: ['https://app.intmax.io/bridge'],
     },
   },
-  escrows: [
+  config: {
+    escrows: [
+      discovery.getEscrowDetails({
+        address: EthereumAddress('0xF65e73aAc9182e353600a916a6c7681F810f79C3'),
+        tokens: '*',
+      }),
+    ],
+  },
+  dataAvailability: {
+    layer: DA_LAYERS.SELF_CUSTODIED_INTMAX,
+    bridge: DA_BRIDGES.SELF_ATTESTED_INTMAX,
+    mode: DA_MODES.BALANCE_PROOF,
+  },
+  technology: {
+    dataAvailability: {
+      name: 'All data required for payments is self-custodied by users',
+      description:
+        'INTMAX uses a self-custodied data availability model where users maintain their own "balance proofs" to allow for private payments. These balance proofs are computed using data received from aggregators when depositing or initiating a transfer, and from payment senders when receiving funds. The protocol ensures that all data has been made available by requiring users to sign off blocks that contain their deposits or outgoing transfers. Users would not accept payments if they have not received the necessary balance proof from the sender.',
+      references: [
+        {
+          title: 'INTMAX Whitepaper',
+          url: 'https://eprint.iacr.org/2023/1082.pdf',
+        },
+      ],
+      risks: [
+        {
+          category: 'Funds can be lost if',
+          text: 'users lose the self-custodied data required to prove their balance.',
+        },
+      ],
+    },
+    operator: {
+      ...OPERATOR.DECENTRALIZED_OPERATOR,
+      references: [
+        {
+          title: 'INTMAX Block Builder - INTMAX docs',
+          url: 'https://intmax-wallet.gitbook.io/intmax-developers-hub/intmax-block-builder',
+        },
+      ],
+    },
+    exitMechanisms: [
+      {
+        name: 'Users can autonomously exit their funds',
+        description:
+          'Users can autonomously exit by providing a ZK proof of sufficient balance. This requires keeping track of all funds received and sent.',
```

Comment:
I would stress that the locked tokens are on Ethereum L1 (in Liquidity SC) since this is something Intmax highlight several times in their docs. I think they don't want to be seen as a full L3 because of this and it makes sense.

I think this section could be one fitting place, but maybe we can mention it somewhere else as well